airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml

```diff
@@ -15,18 +15,20 @@ tests:
       status: "exception"
   discovery:
     - config_path: "secrets/config.json"
+      backward_compatibility_tests_config:
+        disable_for_version: "0.1.83"
   basic_read:
     - config_path: "secrets/config.json"
       timeout_seconds: 600
       configured_catalog_path: "sample_files/basic_read_catalog.json"
-      empty_streams: ["form_submissions", "ticket_pipelines", "engagements_meetings", "engagements_emails", "engagements", "feedback_submissions", "engagements_calls", "quotes"]
+      empty_streams: ["form_submissions", "ticket_pipelines", "engagements_meetings", "engagements_emails", "engagements", "engagements_calls", "quotes"]
```

Comment:
@YowanR, these streams are empty on our sandbox account it means they are not tested properly covered by SAT and might not work as expected in production. Do you think you could create and prioritize the provisionning of our test account with some data on this stream so we can keep the wolf from the door 😄